sys-kernel/gentoo-kernel/metadata.xml

```diff
@@ -6,6 +6,7 @@
 		<name>Distribution Kernel Project</name>
 	</maintainer>
 	<use>
+		<flag name="generic-uki">Build and install a generic initramfs and Unified Kernel Image (EXPERIMENTAL)</flag>
```

Comment:
Maybe we should clarify why someone might want this for the non-bin?

Reply:
> Maybe we should clarify why someone might want this for the non-bin?

To be honest, I don't really see why anyone would want to enable this flag except specifically for building binpkgs.

Reply:
Maybe just add "This is intended for building binpkgs."

Reply:
Added.